evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -320,6 +321,26 @@ impl<'a, B: Backend> CheatcodeStackExecutor<'a, B> {
 
         Capture::Exit((ExitReason::Succeed(ExitSucceed::Stopped), res))
     }
+    fn console_log(&mut self, input: Vec<u8>) -> Capture<(ExitReason, Vec<u8>), Infallible> {
+        let decoded = match ConsoleLogCalls::decode(&input) {
+            Ok(inner) => inner,
+            Err(err) => return evm_error(&err.to_string()),
+        };
+        match decoded {
```

Comment:
I think you should not even need to match? Just println!("{}", decoded)

Reply:
I need to match to get access to the `inner.0` which has the actual string. 

let me try with println! too

Reply:
All the outer structs/enum impl Display so it should just work